cli/cmd/validate_config_test.go

```diff
@@ -0,0 +1,57 @@
+package cmd
+
+import (
+	"os"
+	"path"
+	"runtime"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestValidateConfig(t *testing.T) {
+	configs := []struct {
+		name   string
+		config string
+		errors []string
+	}{
+		{
+			name:   "multiple test sources should pass validation",
+			config: "multiple-sources.yml",
+		},
+		{
+			name:   "bad AWS and Postgres auth should fail validation",
+			config: "validate-config-error.yml",
+			errors: []string{"failed to validate source config aws", "failed to validate destination config postgresql"},
+		},
+	}
+	_, filename, _, _ := runtime.Caller(0)
+	currentDir := path.Dir(filename)
+	cqDir := t.TempDir()
+	defer os.RemoveAll(cqDir)
+
+	for _, tc := range configs {
+		t.Run(tc.name, func(t *testing.T) {
+			defer CloseLogFile()
+			testConfig := path.Join(currentDir, "testdata", tc.config)
+			logFileName := path.Join(cqDir, "cloudquery.log")
+			cmd := NewCmdRoot()
+			cmd.SetArgs([]string{"validate-config", testConfig, "--cq-dir", cqDir, "--log-file-name", logFileName})
```

Comment:
Let's use https://github.com/cloudquery/cloudquery/blob/f509e8b1c961427979cdb3a525e7bc88bdaea308/cli/cmd/cmd_test.go#L8 here to ensure different tests share the log file/cq dir